app/src/main/java/org/oppia/android/app/settings/profile/SnackbarController.kt

```diff
@@ -0,0 +1,39 @@
+package org.oppia.android.app.settings.profile
+
+import androidx.annotation.StringRes
+import org.oppia.android.util.data.DataProvider
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+abstract class SnackbarController @Inject constructor() {
```

Comment:
Add a `DataProviders` constructor parameter here that will help you return a DataProvider object in the `getCurrentSnackbar()` method. Also make sure to include an import for it. Here is what your class declaration should look like;

    import org.oppia.android.util.data.DataProviders
    
    @Singleton
    class SnackbarController @Inject constructor(
        private val dataProviders: DataProviders
    ) {...}

Reply:
Okay